packages/javascript/src/span.ts

```diff
@@ -60,7 +60,7 @@ export class Span extends Serializable<JSSpanData> {
     return this
   }
 
-  public setParams(params: HashMap<HashMapValue>): this {
+  public setParams(params: HashMap<any>): this {
     this._data.params = { ...this._data.params, ...params }
```

Comment:
Huh, today I learned. The params being merged is not how the params work in any other integrations... 🙃